packages/babel-helpers/src/helpers.ts

```diff
@@ -1935,3 +1935,10 @@ helpers.identity = helper("7.17.0")`
     return x;
   }
 `;
+
+helpers.setFunctionName = helper("7.20.1")`
+  export default function _setFunctionName(f, n) {
+    if (typeof n === "symbol") n = n.description ? "[" + n.description + "]" : "";
+    return Object.defineProperty(f, "name", { configurable: true, value: n + "" });
```

Comment:
In some older browsers `.name` was non-configurable, so we should wrap this in try/catch.